src/mixins/style-propable.js

```diff
@@ -20,14 +20,8 @@ export default {
     return ImmutabilityHelper.merge.apply(this, arguments);
   },
 
-  //Moved this function to /utils/styles.js
-  mergeAndPrefix() {
```

Comment:
:crying_cat_face: Yeah you're probably right. This happened recently in #2672 when someone was attempting to use it. I'm getting trigger happy to reduce our API surface. I don't mind doing a deprecation until v0.15.0 though, I think that's fair.